### ISSUE TITLE
Contacto emergente de una evolución o de un giro

### ISSUE THREAD

**Issue by jmlahoz** [agarsev/signotator#13]
A la luz del signo DICCIONARIO, ¿sigues creyendo que no puede haber contacto si no hay desplazamiento?

**Comment by agarsev** [comment]
Pues claramente hay que promocionar el rasgo contacto como independiente del lugar, ya son demasiados ejemplos. En signotación no hay problema, entiendo, pero el signotador sí que tiene entonces que ser adaptado.

**Comment by jmlahoz** [comment]
Sí, podría ser que en origen esté vinculado al lugar (y en los segmentos estáticos ---coda o ataque--- así es), pero que cualquier dinamicidad en el fondo modifique la manera de estar la mano en el espacio y, por tanto, pueda generar contactos de manera emergente.

Por otro lado, los signos con articulador antebrazo pueden aportar un argumento a favor de que los giros lleven contactos aparejados. Por ejemplo, ALASKA, si se describiera como movimiento de la mano sería un arco (un D, al que no pondríamos reparo en añadir un contacto), pero si es más bien un movimiento de antebrazo sería un giro % (y tendría que seguir llevando el contacto).
